app/ide-desktop/lib/dashboard/src/utilities/HttpClient.ts

```diff
@@ -82,7 +88,8 @@ export default class HttpClient {
     method: HttpMethod,
     url: string,
     payload?: BodyInit,
-    mimetype?: string
+    mimetype?: string,
```

Comment:
Oh, can we pass the mimetype and keepalive as fields of options object? What if I want pass keepalive only?